src/Factory/SwooleEventListenerDelegator.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Factory;
+
+use Mezzio\Swoole\Exception\InvalidListenerException;
+use Phly\EventDispatcher\ListenerProvider\AttachableListenerProvider;
+use Psr\Container\ContainerInterface;
+use Webmozart\Assert\Assert;
+
+use function is_callable;
+use function is_string;
+use function sprintf;
+
+class SwooleEventListenerDelegator
```

Comment:
Ideally, I'd like to extract the functionality that loops through the mezzio-swoole listeners and resolves them to services into a class in the mezzio-swoole package, as this is tedious to do when you want to have task listeners.